docs/source/custom/package.rst

```diff
@@ -5,7 +5,8 @@ Using and Distributing Extensions
 Extensions for :py:mod:`cobald` are regular Python code accessible to the interpreter.
 For specific problems, extensions can be defined directly in a Python configuration file.
 General purpose and reusable code should be made available as a Python package.
-This ensures proper installation and dependency management.
+This ensures proper installation and dependency management,
+and allows quick access in YAML configuration files.
```

Comment:
This is a bit confusing. I do see two options here what *in YAML configuration files* mean:

* in terms of *from YAML configuration files*, aka from within a configuration you access the Python package, or
* in terms of *to YAML configuration files*, aka from Python to the YAML configuration.

I think it might be the first point.

```suggestion
and allows quick access from YAML configuration files.
```